src/ds/champ_map.h

```diff
@@ -165,6 +182,8 @@ namespace champ
   template <class K, class V, class H>
   struct SubNodes
   {
+    // TODO: Reduce node size by replacing vector with `Node<K, V, H>*`, since
+    // the length is equal to `node_map.pop() + data_map.pop()`
```

Comment:
We currently have a policy of 0 TODOs in the code. If this is a small addition, please add it to this PR, otherwise please create a new GitHub issue with the `TODO` tag to track this.

Reply:
Issue opened: https://github.com/microsoft/CCF/issues/1661, and TODO comment removed